index.html

```diff
@@ -1,214 +1,212 @@
+        <div class="container-fluid"><a class="navbar-brand" href="#"><img src="/assets/img/fern128.png" style="width: 50px;"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon"></span></button>
+            <div
+                class="collapse navbar-collapse" id="navcol-1" style="justify-content: flex-end;">
+                <ul class="nav navbar-nav nav-list">
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="#usage">Usage</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="#backup">Backup &amp; Restore</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="#limit">Limitations</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="#related">Privacy policy</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="https://github.com/cxw42/TabFern">View on Github</a></li>
+                    <li class="nav-item" role="presentation"><a class="nav-link" href="https://chrome.google.com/webstore/detail/tabfern-tab-manager-and-b/hbajjpcdbninabigakflkhiogmmjaakm">Add to Chrome</a></li>
+                </ul>
+        </div>
+        </div>
+    </nav>
+    <div style="min-width: 100vw;font-family: 'Josefin Sans', sans-serif;background-color: #4C8BF5;">
+        <div class="row">
+            <div class="col">
+                <div style="display: flex;flex-direction: column;padding: 20px 5px;">
+                    <header style="margin: auto;padding: 20px 5px;">
+                        <h1 style="font-size: 80px;text-align: center;color: white;font-weight: 800;margin: 60px 0px;">TabFern</h1>
+                        <p style="text-align: center;font-weight: 100;font-size: 22px;padding: 10px;word-break: break-all;color: #000000;margin-top: 40px;">Google Chrome extension for saving and restoring sets of tabs, and for switching between windows and tabs from a vertical, grouped list.</p>
+                    </header>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div>
+        <div class="container" style="padding: 0px;max-width: 1400px;">
+            <div>
+                <div class="row rower">
+                    <div class="col-md-6">
+                        <div style="padding: 50px 10px;display: flex;flex-direction: column;">
+                            <header style="justify-content: flex-start;justify-self: flex-start;">
+                                <h1 style="font-size: 64px;">What we do?</h1>
+                            </header>
+                            <ul class="text-capitalize what-we-do" style="margin: auto;margin-top: 20px;list-style-image: url(&quot;/assets/img/bullet.png&quot;);width: 100%;">
```

Comment:
Please remove `text-capitalize` everywhere it is used.  This is for my personal ease of reading :) .